doc/filters.texi

```diff
@@ -10310,6 +10310,11 @@ The x and y offset coordinates where the box is drawn.
 @item h
 The width and height of the drawn box.
```

Comment:
it is not correct for face-detection-adas-0001.xml:
input=data:output=detection_out

Reply:
sorry, i  didn't get it, i just used the parameter same as you

Reply:
my fault, i made a mistake.